coverage/src/test/kotlin/org/rust/coverage/RsCoverageTest.kt

```diff
@@ -0,0 +1,222 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.coverage
+
+import com.intellij.execution.configurations.RunConfiguration
+import com.intellij.execution.executors.DefaultRunExecutor
+import com.intellij.execution.runners.ExecutionEnvironmentBuilder
+import com.intellij.openapi.vfs.VfsUtil
+import com.intellij.rt.coverage.data.ClassData
+import com.intellij.rt.coverage.data.LineData
+import org.rust.FileTreeBuilder
+import org.rust.cargo.runconfig.CargoRunStateBase
+import org.rust.coverage.RsCoverageRunner.Companion.lastProjectData
+import org.rust.lang.core.psi.isRustFile
+import org.rust.openapiext.toPsiFile
+import org.rustSlowTests.cargo.runconfig.RunConfigurationTestBase
+
+class RsCoverageTest : RunConfigurationTestBase() {
+
+    override fun setUp() {
+        super.setUp()
+        lastProjectData = null
+    }
+
+    override fun tearDown() {
+        super.tearDown()
+        lastProjectData = null
```

Comment:
In this case, it shouldn't affect anything if everything is ok, but you should clear your state before `super.tearDown()`